src/Cli/dotnet/commands/dotnet-workload/config/WorkloadConfigCommand.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Deployment.DotNet.Releases;
+using Microsoft.DotNet.Cli;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Workloads.Workload;
+using Microsoft.DotNet.Workloads.Workload.Install;
+using Microsoft.NET.Sdk.WorkloadManifestReader;
+
+#nullable enable
+
+namespace Microsoft.DotNet.Workloads.Workload.Config
+{
+    internal class WorkloadConfigCommand : WorkloadCommandBase
+    {
+        bool _hasUpdateMode;
+        string? _updateMode;
```

Comment:
nit `private`

Reply:
`private` would be redundant here.  I know it's inconsistent here and probably is throughout our codebase.  Do we have a convention we are trying to follow for whether we should include `private` or not?

Reply:
I generally prefer including it...not technically an answer, but it's my preference :)

Reply:
I'm pretty sure explicit access modifiers for fields are the convention in the dotnet org. I'm surprised we don't have that enabled in our editorconfig.